upgrade_analysis/tests/test_module.py

```diff
@@ -18,19 +18,19 @@ def test_upgrade_install_wizard(self):
 
         wizard.select_odoo_modules()
         self.assertTrue(
-            self.product_module.id in wizard.module_ids.ids,
+            self.website_module.id in wizard.module_ids.ids,
             "Select Odoo module should select 'product' module",
         )
-
-        wizard.select_oca_modules()
-        self.assertTrue(
-            self.upgrade_analysis.id in wizard.module_ids.ids,
-            "Select OCA module should select 'upgrade_analysis' module",
-        )
+        # New patch avoids to reinstall already installed modules, so this will fail
```

Comment:
OK, you can remove this commented code given that reinstalling seems save.

Reply:
Done @StefanRijnhart